src/app/auth/LoginComponent.tsx

```diff
@@ -170,4 +178,14 @@ class LoginComponent extends React.Component<any, any> {
     );
   }
 }
-export default LoginComponent;
+
```

Comment:
I dont really see a reason to move this here. Since we want to limit the amount of components that know about the redux store, this is only happening in 3 or 4 components anyway. Here is an article about the benefits of separating this logic out into its own container. It is a common pattern. https://www.fullstackreact.com/p/using-presentational-and-container-components-with-redux/

Reply:
Yes, but this article describe benefits in case generic components, `LogicComponent` is not generic. So as wrote previously: But it could work in cases when component is used separately from it 'connected' version, this is not that case, so yeah, I prefer this pattern.

Reply:
ok thats no problem. we can combine for now & seperate out later if we need to.